src/libraries/System.Security.Cryptography.Pkcs/src/System/Security/Cryptography/Pkcs/Rfc3161TimestampRequest.cs

```diff
@@ -38,7 +38,7 @@ public X509ExtensionCollection GetExtensions()
                 return coll;
             }
 
-            X509ExtensionAsn[] rawExtensions = _parsedData.Extensions;
+            X509ExtensionAsn[] rawExtensions = _parsedData.Extensions!;
```

Comment:
The `if (!HasExtensions)` on line 36 checks that `Extensions` is not null and > 0.